src/Microsoft.PowerShell.Commands.Utility/commands/utility/write.cs

```diff
@@ -238,35 +232,34 @@ public class WriteOrThrowErrorCommand : PSCmdlet
 
         /// <summary>
         /// If Exception is specified, this is ErrorRecord.ErrorDetails.Message.
-        /// Otherwise, the Exception is System.Exception, and this is
-        /// Exception.Message.
+        /// Otherwise, the Exception is System.Exception, and this is Exception.Message.
```

Comment:
Suggest the same as above....   ErrorDetails.Message; otherwise, ...

Reply:
Fixed.